doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -129,6 +133,18 @@ When you specify the bridge network IP range, you specify the IP range as a CIDR
 
 <pre>--bridge-network-range 192.168.100.0/24</pre>
 
+### `dns-server` ###
+
+Short name: None
+
+A DNS server to use if you specify static IP addresses for the virtual container host on the client, external, and management networks. You can specify `dns-server` multiple times, to configure multiple DNS servers.  
+
+If you specify `dns-server` but you do not specify a static IP addresses for one or more of the client, external, and management networks, `vic-machine create` ignores the `dns-server` setting for that network and uses the DNS servers that are provided by DHCP. You can therefore use a mixture of static and DHCP addresses for the virtual container host on the different networks.
```

Comment:
If you specify `dns-server` but you do not specify a static IP addresses for one or more of the client, external, and management networks, `vic-machine create` ignores the `dns-server` setting and uses the DNS servers that are provided by DHCP. 

If you use a mixture of static and DHCP addresses for the virtual container host on the different networks, the virtual container host will use the DNS servers specified by `dns-server` and those provided by DHCP.